phpBB/phpbb/install/helper/iohandler/ajax_iohandler.php

```diff
@@ -90,12 +90,12 @@ class ajax_iohandler extends iohandler_base
 	 * Constructor
 	 *
 	 * @param path_helper						$path_helper
-	 * @param \phpbb\request\request_interface	$request	HTTP request interface
+	 * @param \phpbb\request\request	$request	HTTP request interface
 	 * @param \phpbb\template\template			$template	Template engine
 	 * @param router 							$router		Router
 	 * @param string 							$root_path	Path to phpBB's root
 	 */
-	public function __construct(path_helper $path_helper, \phpbb\request\request_interface $request, \phpbb\template\template $template, router $router, $root_path)
+	public function __construct(path_helper $path_helper, \phpbb\request\request $request, \phpbb\template\template $template, router $router, $root_path)
```

Comment:
The `raw_variable` method is only part of the request class but not of the request_interface. This was a conscious choice as some other variables of request also are not part of the request_interface, mostly ones that are usually not required for a normal page operation (which IMHO also applies to raw input variables).